File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
class Solution {
    public int[] solution(int[] array) {
        int answer = 0;
        int index = 0;
        
        for(int i=0; i<array.length; i++) {
            if(array[i] > answer) {
                answer = array[i];
                index = i;
            }
        }
        
        int[] ans = new int[2];
        ans[0] = answer;
        ans[1] = index;
        
        return ans;
    }
}

File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.js
function solution(array) {
//     let answer = [];
//     let max = array[0];
    
//     for(let i=1; i<array.length; i++) {
//         if(max < array[i]) {
//             max = array[i];
//             index = i;
//         } 
//     }
    
//     answer.push(max);
//     answer.push(index);
    
//     return answer;
    
    return [Math.max(...array), array.indexOf(Math.max(...array))];
}